shell/platform/android/test/io/flutter/embedding/android/FlutterActivityAndFragmentDelegateTest.java

```diff
@@ -308,7 +308,7 @@ public void flutterEngineGroupGetsInitialRouteFromIntent() {
     ArgumentCaptor<FlutterEngineGroup.Options> optionsCaptor =
         ArgumentCaptor.forClass(FlutterEngineGroup.Options.class);
     verify(flutterEngineGroup, times(1)).createAndRunEngine(optionsCaptor.capture());
-    assertEquals("/initial_route", optionsCaptor.getValue().getInitialRoute());
+    assertEquals("foo://example.com/initial_route", optionsCaptor.getValue().getInitialRoute());
```

Comment:
When these strings to go the framework, are they handled as dart `Uri` objects? If so this is probably fine, but if not won't this cause breakages in framework apps?

Even if so, won't this cause breakages to people using `toString` on the Uri?

Reply:
yes the framework is treated it as uri object after https://github.com/flutter/flutter/pull/119968

Reply:
> Even if so, won't this cause breakages to people using toString on the Uri?

I am not sure if i understand, can you add more details?